app/components/Pagination.tsx

```diff
@@ -14,13 +14,7 @@ import {
 
 const Tunnel = tunnel('pagination')
 
-interface PaginationProps extends UIPaginationProps {
-  /** If true pagination will be rendered wherever `Pagination.Target` is included */
-  inline?: boolean
-}
-export function Pagination({ inline = false, ...props }: PaginationProps) {
-  if (inline) return <UIPagination {...props} />
-
+export function Pagination(props: UIPaginationProps) {
```

Comment:
`inline` was never true here. The only callsite is in `QueryTable`, which defaults to `page` and was never overridden.